pages/1.13/administering-clusters/package-registry/index.md

```diff
@@ -275,7 +277,7 @@ dcos registry migrate --package-directory=/path/to/package --output-directory=/p
 dcos registry build --build-definition-file=/path/to/output/<json-build-defintion-generated-above> --output-directory=/path/to/output
 ```
 
-If all the above steps are completed successfully your `/path/to/output` directory should look similar to:
+If all the above steps are completed successfully, your `/path/to/output` directory should look similar to:
```

Comment:
Rewrite `If all the above steps are completed successfully, your `/path/to/output` directory should look similar to:` with `If all the above steps are completed successfully, your `/path/to/output` directory should look similar to the following:`

Reply:
Done.